consensus/types/round_state.go

```diff
@@ -135,11 +135,11 @@ func (rs *RoundState) NewRoundEvent() types.EventDataNewRound {
 
 	return types.EventDataNewRound{
 		Height: rs.Height,
-		Round:  rs.Round,
+		Round:  int(rs.Round),
```

Comment:
this is needed since we in the past encoded and decoded ints as strings, if we were to change this to `1` instead of `"1"` then I would be able to remove these conversions

Reply:
undid this in favor of having round represented not as a string in json